manifests/bosh-manifest/deployments/040-collectd.yml

```diff
@@ -0,0 +1,56 @@
+---
+releases:
+- name: collectd
+  sha1: 5e5c196f9c95fe12eb17bba0f40fdaecaddd6424
+  url: https://github.com/alphagov/paas-collectd-boshrelease/releases/download/0.4/collectd-0.4.tgz
```

Comment:
The initial reason why we used fork was that we could add custom tags and pin our deployment to these tags.
Now, because bosh-init can only consume tarballs (you can't upload a release to bosh-init like you can to BOSH), I had to create a dev-release tarbal (using standard bosh-cli way `bosh create release --with-tarball --force --version 0.4`, more documentation about bosh releases: https://bosh.io/docs/create-release.html#dev-release-release). I have created the tarballs from the same code as is tagged. That is, 0.4 release is created from code that's tagged as 0.4.

It seemed to me that a good place to put the release tarball was in github release. This practice is also used in some CF repositories (e.g. diego). We have a spike to analyse our situation with releases completely, but we did not want to block this story on it (#115142265 - see comments in the story). In the future, we can easily move the release location to elsewhere and we can change the way to build the releases.

Regarding changing the commit messages:
* I think it should be instead described in the repo why we have a fork of it (e.g. in readme). It seems a bit odd to have to search through commits in other repos hoping for explanation.
* steps to create and upload release are a standard thing, documented on BOSH side or when you print help of bosh-cli. If we have a special workflow (e.g. where to put the releases), they should be described in ADR or team manual or some other generally accessible place, instead of random commit messages.
* both information above are not really related to the code in the commit